backend/tests/integration/testutils/common.py

```diff
@@ -647,7 +647,7 @@ def request_get_all(self, path):
 def retry(
     period: timedelta = timedelta(milliseconds=100),
     timeout: timedelta = timedelta(minutes=2),
-) -> Generator[datetime]:
+) -> Generator[datetime, None, None]:
```

Comment:
Shouldn't it be like this?
```suggestion
) -> Generator[datetime | None]:
```

Reply:
Sorry, should've consulted the documentation first :see_no_evil:

Reply:
I probably should've just used:
```suggestion
) -> Iterator[datetime]:
```